specification/advisor/resource-manager/Microsoft.Advisor/stable/2017-04-19/advisor.json

```diff
@@ -765,6 +749,11 @@
       "description": "The Advisor configuration data structure.",
       "type": "object",
       "properties": {
```

Comment:
type property missing in the response. the value will be Microsoft.Advisor/configurations